### ISSUE TITLE
Convert concatinated strings to f-strings

### ISSUE THREAD

**Issue by mikeckennedy** [ikamensh/flynt#14]
If you take the string:

```
t = 'T is a string of value: ' + val + ' and that's great!'
```

That would be better as an f-string I think. Definitely less error prone (e.g. val not a `str`) . What do you think? Make it:

```
t = f'T is a string of value: {val} and that's great!'
```

**Comment by ikamensh** [comment]
Good proposal, though I would make this transformation optional - some projects might actually rely on exception being thrown. It is not easy to implement, I hope to give it a shot within a month.

**Comment by mikeckennedy** [comment]
Ok, thanks for considering it!

**Comment by ikamensh** [comment]
Good news - this is implemented in flynt 0.35. There is a restriction - flynt has to be installed in a python 3.8+ interpreter. This is because 3.8 adds functionality (end_lineno, end_col_offset) to `ast` module, which is needed to locate candidates for the transformation.

Would be happy to hear back if you could try it on some code. I have tried it on flask, it transformed 12 string concats, and tests were passing afterwards.

**Comment by mikeckennedy** [comment]
Sweet! So good to hear. I'll definitely let you know how it goes when I get Python 3.8 installed (still doing everything on 3.7). Love Flynt, thanks.
